noxfile.py

```diff
@@ -265,27 +276,6 @@ def lint_setup_py(session):
     session.run("python", "setup.py", "check", "--restructuredtext", "--strict")
 
 
-@nox.session(python=DEFAULT_PYTHON_VERSION)
-def pytype(session):
-    """Run type-checking."""
-    session.install(".[grpc]", "pytype")
-    session.run("pytype")
-
-
-@nox.session(python=DEFAULT_PYTHON_VERSION)
-def mypy(session):
-    """Run type-checking."""
-    session.install(".[grpc,async_rest]", "mypy")
-    session.install(
-        "types-setuptools",
-        "types-requests",
-        "types-protobuf",
-        "types-dataclasses",
-        "types-mock; python_version=='3.7'",
-    )
-    session.run("mypy", "google", "tests")
```

Comment:
I do think there's value in being able to run mypy on its own sometimes. 

It could be annoying to have to fix all other style issues, before even seeing if the types are valid

Reply:
Ack. I'll keep mypy as a separate presubmit